src/libp2p_yamux_session.erl

```diff
@@ -155,7 +155,15 @@ handle_info({inert_read, _, _}, State=#state{connection=Connection}) ->
             lager:notice("session closed"),
             {stop, normal, State};
         {error, Reason} ->
-            lager:error("Session header read failed: ~p ", [Reason]),
+            %% Apparently this happens a LOT
+            %% just flag a notice, don't pollute the logs with an error
+            case Reason == enotconn of
```

Comment:
just do `{error, enotconn} -> ` and a next clause with `{error, reason}`. No need to nest case and have a boolean case statement  like this

Reply:
Addresses: a07238d